02. 렌더링/02/view/utils.js

```diff
@@ -0,0 +1,13 @@
+export const createInnerHTML = (tagName, { attributes = {}, innerHTML = '' } = {}) => {
+  const arrtibuteString = Object.entries(attributes)
+    .map(([name, value]) => {
+      if (typeof value === 'boolean') return value ? name : '';
+      return `${name}="${value}"`;
+    })
+    .join(' ');
+  const closeTagString = innerHTML !== '' ? `${innerHTML}</${tagName}>` : '';
+
+  return `<${tagName} ${arrtibuteString}>${closeTagString}`;
```

Comment:
```suggestion
  return `<${tagName} ${arrtibuteString}>${innerHTML}</${tagName}>`;
```
`innerHTML` 매개변수 기본값으로 빈 문자열 넣어주니까 `closeTagString` 변수 따로 만들 필요 없이 이렇게 작성해도 되지 않을까요?